benchmarks/argsort.py

```diff
@@ -13,8 +13,13 @@ def time_ak_argsort(N_per_locale, trials, dtype, seed):
     print("numLocales = {}, N = {:,}".format(cfg["numLocales"], N))
     if dtype == 'int64':
         a = ak.randint(0, 2**32, N, seed=seed)
+        nbytes = a.size * a.itemsize
     elif dtype == 'float64':
         a = ak.randint(0, 1, N, dtype=ak.float64, seed=seed)
+        nbytes = a.size * a.itemsize
+    elif dtype == 'str':
+        a = ak.random_strings_uniform(1, 8, N, seed=seed)
+        nbytes = (a.offsets.size * a.offsets.itemsize) + (a.bytes.size * a.bytes.itemsize)
```

Comment:
Should offsets be left off here? It's technically part of the underlying implementation, but based on the discussion in https://github.com/mhmerrill/arkouda/issues/531#issuecomment-712370644 I wonder if it makes more sense to just track given "X GiB of string data from a user, how fast does that sort" and ignore how we represent/manipulate it internally

Reply:
Good point. I removed offsets.